Codeinwp/templates-patterns-collection
# TPC templates button

### Summary
Added a new button to import a TPC template directly into a post, page, or custom layout

### Test instructions
- Go to a post, a page, or a custom layout, and you should be able to see that button
- The button is available only for PRO users, so make sure you activate the license
- Check if the import is working correctly. If you import more than one template, the adding should be done at the end.

### Screenshots or videos
https://vertis.d.pr/v/uowrr1


Closes Codeinwp/neve-pro-addon#2564.
